locations/name_suggestion_index.py

```diff
@@ -72,14 +72,19 @@ def get_wikidata_code_from_url(self, url: str = None) -> str | None:
                     return wikidata_code
         return None
 
-    def lookup_wikidata(self, wikidata_code: str = None) -> dict:
+    def lookup_wikidata(self, wikidata_code: str = None, include_dissolved=False) -> dict | None:
```

Comment:
Include doc for what the new param means.

Reply:
Done